content/graphql/subscriptions.md

```diff
@@ -186,7 +216,7 @@ With this, we've created a single `commentAdded(title: String!): Comment` subscr
 
 #### PubSub
 
-We used a local `PubSub` instance above. The preferred approach is to define `PubSub` as a [provider](/fundamentals/custom-providers) and inject it through the constructor (using the `@Inject()` decorator). This allows us to re-use the instance across the whole application.
+We instantiated a local `PubSub` instance above. The preferred approach is to define `PubSub` as a [provider](/fundamentals/custom-providers) and inject it through the constructor (using the `@Inject()` decorator). This allows us to re-use the instance across the whole application. For example, define a provider as follows, then inject `'PUB_SUB'` where needed.
 
```

Comment:
Actually, I think we could even simply pass `PubSub` as a provider (unless one wants to pass options):
```
providers: [PubSub]
```
and inject it in a normal way

Reply:
OK, I tested this and see that it *does* work, but I'm slightly confused as to *why*.  I'll leave it like this for now (it's actually essentially what was there before with a few additional clarifying words) until I understand better why the option you suggest works.

I'll leave it up to you whether to merge these changes before/after resolving this one.

Reply:
@johnbiundo using **any** class which doesn't have required constructor arguments as a provider should work in Nest :)

Reply:
Thanks.  Huh. Learn something every day :)

If you're OK with it for now, I'd like to leave as is, as I think it might be confusing to slip in this concept without more context.  I'm thinking about whether we need to explain this concept in the providers chapter.

Reply:
Sure, make sense :)